GetSchwifty/View Controllers/ProjectTableViewController.swift

```diff
@@ -0,0 +1,111 @@
+//
+//  ProjectTableViewController.swift
+//  GetSchwifty
+//
+//  Created by Marius Landwehr on 19.04.17.
+//  Copyright © 2017 Benjamin Herzog. All rights reserved.
+//
+
+import UIKit
+
+protocol ProjectTableViewControllerDelegate: class {
+    func projectTableViewControllerDidPressBack()
+    func projectTableViewControllerDidSelect(schwifty: Schwifty)
+}
+
+class ProjectTableViewController: UITableViewController {
+    
+    private var projects: [[Schwifty]] = [[], []]
+    private let dataStore = SchwiftyDataStorage()
+
+    weak var delegate: ProjectTableViewControllerDelegate?
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        tableView.register(UITableViewCell.self, forCellReuseIdentifier: "ProjectCell")
+        title = "Get Schwifty"
+        tableView.backgroundColor = UIColor(r: 31, g: 32, b: 41, a: 1)
+        tableView.tintColor = UIColor(r: 237, g: 82, b: 63, a: 1)
+        tableView.tableFooterView = UIView()
+        
+        let backButton = UIBarButtonItem(title: "Back", style: .plain, target: self, action: #selector(back))
+        let new = UIBarButtonItem(barButtonSystemItem: .compose, target: self, action: #selector(self.newSchwifty))
+        let info = UIBarButtonItem(image: #imageLiteral(resourceName: "Info"), style: .plain, target: self, action: #selector(showInfo))
+        self.navigationItem.rightBarButtonItems = [new, info]
+        self.navigationItem.leftBarButtonItem = backButton
+    }
+    
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        
+        let all = dataStore.all()
+        let temporary = all.filter { $0.temporary }
+        let solid = all.filter { !$0.temporary }
+        self.projects = [temporary, solid]
+        self.tableView.reloadData()
+    }
+    
+    func back() {
+        self.delegate?.projectTableViewControllerDidPressBack()
+    }
+    
+    func showInfo() {
+        let info = InfoViewController().wrapInNavigationController()
+        self.present(info, animated: true)
+    }
+
+    public func newSchwifty() {
+        self.delegate?.projectTableViewControllerDidSelect(schwifty: Schwifty(source: ""))
+    }
+    
+    // MARK: - Table view data source
+    
+    override func numberOfSections(in tableView: UITableView) -> Int {
+        return 2
```

Comment:
If we would like to hide the section completly if empty, yes. But this is not absolutely needed in my eyes.